Casks/s/simplysign.rb

```diff
@@ -0,0 +1,45 @@
+cask "simplysign" do
+  version "2.10.19-9.3.3.0,2.10.19-9.3.3"
+  sha256 "1c6ccc48ecf5e57b0cd85771ac5209b8d91351268d5302accce034a6b1039cc8"
+
+  url "https://files.certum.eu/software/SimplySignDesktop/OSX/#{version.csv.first}/SimplySignDesktop-#{version.csv.second}_prod.dmg"
```

Comment:
```suggestion
  url "https://files.certum.eu/software/SimplySignDesktop/OSX/#{version.csv.first}/SimplySignDesktop-#{version.csv.second || version.csv.first}_prod.dmg"
```

Per the previous comment, this adapts the `url` based on the presence of a second `version` part, so we can use a single `version` when there isn't any difference in the path version and file name version.